src/components/AddColumnBtn/customColumnHelpers.js

```diff
@@ -0,0 +1,187 @@
+import React from 'react'
+import { Icon } from '../Icon'
+import _cloneDeep from 'lodash.clonedeep'
+
+export const ORDERABLE_WINDOW_FN_TYPES = ['RANK']
+
+export const OPERATORS = {
+  CONCAT: {
+    value: 'CONCAT',
+    label: 'Concatenate(...)',
+    js: '',
+  },
+  ADDITION: {
+    value: 'ADD',
+    label: <Icon type='plus' />,
+    icon: 'plus',
```

Comment:
label and icon values seem mismatched here... same with the below, so i am assuming this is intentional?

Reply:
Yes it is intentionally but I'm actually changing this to either have icon OR label. If icon exists, then the react widgets will use it for the <Icon /> type, and if not, it will use the label as plain text.

I put this into autoql-fe-utils now without the <Icon/> part to make it compatible with both react and vanilla